package/CHANGELOG

```diff
@@ -38,6 +38,7 @@ Deprecations
   * 2.0.0
 
 Fixes
+  * Fix ITPParser charge reading (Issue #3419).
```

Comment:
2.0 is released already so this'll need to go in the 2.1 section

Reply:
Should fix this in https://github.com/MDAnalysis/mdanalysis/pull/3425/commits/a69cd8cd3695aea4309a5b800fac384ee301d605